common/api-review/auth.api.md

```diff
@@ -750,10 +750,40 @@ export function signOut(auth: Auth): Promise<void>;
 export interface TotpMultiFactorAssertion extends MultiFactorAssertion {
 }
 
+// @public
+export class TotpMultiFactorGenerator {
+    static assertionForEnrollment(secret: TotpSecret, oneTimePassword: string): TotpMultiFactorAssertion;
+    static assertionForSignIn(enrollmentId: string, oneTimePassword: string): TotpMultiFactorAssertion;
+    // Warning: (ae-forgotten-export) The symbol "FactorId" needs to be exported by the entry point index.d.ts
+    static FACTOR_ID: FactorId_2;
```

Comment:
`FactorId_2` is fishy, the public type might be using the wrong enum

Reply:
This uses FactorId from public_types - https://github.com/firebase/firebase-js-sdk/blob/mfa-totp/packages/auth/src/mfa/assertions/totp.ts#L21, which is using the enum type - https://github.com/firebase/firebase-js-sdk/blob/fe42bacddbd3586b12fdbebccaddd65bf639086c/packages/auth/src/model/public_types.ts#L546

This looks right to me.. maybe we are running into something like - https://github.com/microsoft/rushstack/issues/3581


The other option is to define FACTOR_ID as string in https://github.com/firebase/firebase-js-sdk/blob/fe42bacddbd3586b12fdbebccaddd65bf639086c/packages/auth/src/mfa/assertions/totp.ts#L107, similar to what phone mfa does - https://github.com/firebase/firebase-js-sdk/blob/mfa-totp/packages/auth/src/platform_browser/mfa/assertions/phone.ts#L98

Reply:
Synced offline that we will leave this as-is for now and switch to String if we see the need for it.